packages/styles/src/core/layout/toolbar/_main.scss

```diff
@@ -1,36 +1,39 @@
+$_toolbar-main__size: 13 * $u1;
```

Comment:
SASS variables don't need to follow BEM 😂, and they don't need `_` modifier.

And let's put variables in its own folder because if we put this here we don't allow anyone to override it.

Reply:
Im not sure what will happen with the components if we put the vars in its own folder 🙄